docs/IDENTITIES.md

```diff
@@ -168,7 +172,7 @@ To use it, add it to the list of known identity providers:
 import { addIdentityProvider } from '@orbitdb/core'
 import MyCustomIdentityProvider from 'my-custom-identity-provider'
 addIdentityProvider(MyCustomIdentityProvider)
-const identity = await createIdentity({ id: 'some id', type: 'custom' })
+const identity = await createIdentity({ type: MyCustomIdentityProvider(options) })
```

Comment:
Here `type` should be `provider` (https://github.com/orbitdb/orbitdb/pull/1090/files#diff-8c445a0a80e9ec75a67dd06dcd352e8bfac45b3a9c67d662a65400c253141733R67)